diff --git a/driver/aliases.py b/driver/aliases.py
index 65329ae6c..487cf162b 100644
--- a/driver/aliases.py
+++ b/driver/aliases.py
@@ -146,6 +146,8 @@ ALIASES["seq-opt-lmcut"] = [
 
 PORTFOLIOS = {}
 for portfolio in os.listdir(PORTFOLIO_DIR):
+    if portfolio == '__pycache__':
+        continue
     name, ext = os.path.splitext(portfolio)
     assert ext == ".py", portfolio
     PORTFOLIOS[name.replace("_", "-")] = os.path.join(PORTFOLIO_DIR, portfolio)
